com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/textureset/test/TextureSetLayoutTest.java

```diff
@@ -36,10 +36,10 @@ public class TextureSetLayoutTest {
 
     void assertRect(Layout layout, int i, String id, int index, int x, int y) {
         Rect r = layout.getRectangles().get(i);
-        assertThat(r.id, is(id));
-        assertThat(r.index, is(index));
-        assertThat(r.x, is(x));
-        assertThat(r.y, is(y));
+        assertThat(r.getId(), is(id));
```

Comment:
Jhonny, most of this code has been reviewed previously in a (now closed) PR.
Nothing much has really changed since that PR, just more getters/setters in the TextureSetLayout, and some new editor code.